kronos-java/src/main/java/com/lyft/kronos/internal/ntp/NTPSyncException.kt

```diff
@@ -2,4 +2,4 @@ package com.lyft.kronos.internal.ntp
 
 import java.lang.RuntimeException
 
-class NTPSyncException(message: String) : RuntimeException(message)
\ No newline at end of file
+open class NTPSyncException(message: String) : RuntimeException(message)
```

Comment:
The exception is not subclassed here. Are you sure it needs to be open?

Reply:
oups. my bad. I had subclassed it while I was in development. I will remove it.